docs/modules/ROOT/pages/tmail-backend/jmap-extensions/deletedMessagesVault.adoc

```diff
@@ -44,11 +44,22 @@ It has the following properties:
  - `failed`: the task execution failed.
  - `canceled`: the task was canceled.
 
-=== EmailRecoveryAction/get
+== EmailRecoveryActionResult object
+
+This represents the result of submitted email recovery action.
+
+It has the following properties:
+
+- `id`: *Id* (server set). The object identifier. This identifier is mapped one-to-one to the identifier of the `EmailRecoveryAction` object.
+- `successfulRestoreCount`: *UnsignedInt* (server-set). Number of successful restored messages.
+- `errorRestoreCount`: *UnsignedInt* (server-set). Number of failed restored messages.
+- `status`: *Status* (server-set). The status of the task.
```

Comment:
IMO we do not need ` EmailRecoveryActionResult object`

 let's add `successfulRestoreCount` and `errorRestoreCount` to `EmailRecoveryAction` (server set) + mention explicitly which properties can be used within `EmailRecoveryAction/get` ?